src/test/java/com/exactpro/junit5/WithByteBufParameterResolver.java

```diff
@@ -0,0 +1,35 @@
+package com.exactpro.junit5;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
+
+import java.lang.reflect.Executable;
+import java.lang.reflect.Method;
+
+public class WithByteBufParameterResolver implements ParameterResolver {
+
+    private final WithByteBufMethodContext methodContext;
+
+    private final Object arguments;
+
+    WithByteBufParameterResolver(WithByteBufMethodContext methodContext, Object arguments) {
```

Comment:
I think in our case arguments is always List<ByteBuf> and it should be possible to change code accordingly.